src/DependencyInjection/Configuration.php

```diff
@@ -29,23 +29,24 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->children()
-                ->integerNode('request_limit_per_node')
-                    ->min(0)
-                    ->defaultValue(150)
-                ->end()
-                ->integerNode('request_window_per_node')
-                    ->defaultValue(3600)
-                    ->min(0)
-                ->end()
-                ->integerNode('max_execution_time')
-                    ->min(1)
-                    ->defaultValue(16)
-                ->end()
-                ->enumNode('trigger_type')
-                    ->values(['internal', 'route'])
-                    ->defaultValue('internal')
-                ->end()
-            ->end();
+            ->integerNode('request_limit_per_node')
```

Comment:
@m-vo I'm not sure why the skip of `MethodChainingIndentationFixer` did not work correctly here! Maybe we should use v5 of `contao/easy-coding-standard`